rc-events-service/src/main/scala/com/epam/random_coffee/events/api/request/CreateEventRequest.scala

```diff
@@ -1,3 +1,3 @@
 package com.epam.random_coffee.events.api.request
 
-case class CreateEventRequest(name: String)
+case class CreateEventRequest(eventName: String, description: String, eventDate: String, author: String)
```

Comment:
Again, there is no reason for name fields with prefix "event", when you have that in class name.
Anyway, if you decided to name it like so, you should name description like eventDescription than.

Reply:
Fixed